services/docker-api-proxy/requirements/_test.in

```diff
@@ -0,0 +1,18 @@
+--constraint ../../../requirements/constraints.txt
+
+--requirement ../../../packages/common-library/requirements/_base.in
+--requirement ../../../packages/models-library/requirements/_base.in
+--requirement ../../../packages/service-library/requirements/_base.in
```

Comment:
I think you do not need these dependencies

Reply:
I've checked all the required dependencies added them in. The current list is now correct.

Reply:
When I run the tests without these packages. I get an error saying they are required.